Translated _messages.js

lynparbone opened the pull request:
Translated the complete file appinventor-sources/appinventor/blocklyeditor/src/msg/de/_messages.js from English to German (ZHAW, as discussed with Marina).
Please let us know when the translations are available on a test server so we can verify them in the application.

Commit:
Translations to line 366

Commit:
Translations to line 802

Commit:
Finished translations

Commit:
Small finishing changes

m5muma2 commented:
Hi everybody, we are happy to help. As far as we see you the translation work is done. Great job! Shall we (Johannes and me from Jena) test it or could we help in another way?

ewpatton commented:
I have put up a test server [here](http://1.de.ai2-ewpatton-temp.appspot.com/).

m5muma2 commented:
We found some mistakes in the German version on the test server, which we couldn´t correct here. Please find a list attached.

https://drive.google.com/open?id=1nxNd28-JeHWvZ8r68n6mn7PhoaEyxMpo

m5muma2 committed:
Update OdeMessages_de.properties

I made some changes in the file. I started with the bottom line and stopped in Line 350. The last lines will be done in the next days. I found the suggestions 0, 2, 4, 5, 6, 9, 14, 21, 22, 34, 36, 37, 39, 40, 41, 42, 43, 44, 46, 59, 63, 67, 68, 71, 72, 74, 75, 76 and 77 of our table. Furthermore, the use of the German "Sie" and "Du" is inconsistent in the file. Most time, you use "Sie", but there is at least one message using "Du". Best, Matthias

Commit:
Changed Sie to Du Form, made some small corrections

lynparbone committed:
Merge pull request #1 from m5muma2/patch-1

Update OdeMessages_de.properties
